### PR TITLE
Simplified backend challenge and clarified READMEs

### PR DESCRIPTION
**Pull request by marcgg**

### CODE REVIEW

File: backend/level3/README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Level 3
 
-The car owner now wants his money.
+The car owner now wants her money.
```
Comment (EmilyFiennes) [inline backend/level3/README.md:3]: 💜 
👫
🚀
🌈

Comment (marcgg) [reply backend/level3/README.md:3]: Aha yeah, I've made the owner a woman and the driver a man 🌈 
(hopefully I kept it consistent)

Comment (EmilyFiennes) [reply backend/level3/README.md:3]: I was going to say you could take it a step further and keep it gender neutral with 'Their' but then it gets a bit complicated if they give them their money for example... 🙄

File: backend/level4/README.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+We want to know how much money must be debited/credited for each actor.
 
-The rule is simple: the driver is charged an additional 4€/day when she chooses the "deductible reduction" option.
-**The money goes to us, not to the owner.**
+Reminder:
+
+- The *owner* receives the rental price minus the commission
+- The *insurance* receives its part of the commission
+- The *assistance* receives its part of the commission
+- *Drivy* receives its part of the commission
```
Comment (EmilyFiennes) [inline backend/level4/README.md]: You could make this clearer with 'The insurance provider receives their part of the commission' and 'The assistance provider receives their part of the commission' (and so same [here](https://github.com/drivy/jobs/pull/6/files#diff-f05e4bcf49ea623697df738d73ebfb91R7)

File: backend/level3/README.md (hunk 1 above)
Comment (marcgg) [reply backend/level3/README.md:3]: Yeah it was my first draft, but the gender neutral approach made things quite hard to read. Also a lot of candidates will be french & will not get the nuance.

English needs to figure out a better way to do gender neutral ^^

File: backend/level4/README.md (hunk 2 above)
Comment (marcgg) [reply backend/level4/README.md]: Actually reading it again, I'm not sure it's even worth listing this.

How about just "We want to know how much money must be debited/credited for each actor." and that's it?

Comment (EmilyFiennes) [reply backend/level4/README.md]: 👍

**Pull request merged**